code/datums/wounds/cuts.dm

```diff
@@ -152,6 +152,11 @@
 	user.visible_message("<span class='notice'>[user] licks the wounds on [victim]'s [limb.name].</span>", "<span class='notice'>You lick some of the wounds on [victim]'s [limb.name]</span>", ignored_mobs=victim)
 	to_chat(victim, "<span class='green'>[user] licks the wounds on your [limb.name]!</span")
 	blood_flow -= 0.5
+	if( ismoth(victim) || iscatperson(victim) || ismonkey(victim) ) // Yep you can lick monkeys.
+		user.reagents.add_reagent(/datum/reagent/hairball, 2)
```

Comment:
change ismoth to isinsect, they do the same thing because we merged moth into a insect species, and a species-cleanup pr i have is removing ismoth from the code which could cause an issue here (also remove the first space!)